src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -34,8 +35,13 @@
         @Column(name = "VALUE")
         private BigDecimal value;
 
-        public Cart(Long cartId, BigDecimal value) {
+        public Order getOrder() {
+                return new Order();
+        }
```

Comment:
Co to? 😉 chyba coś niechcący się zaplątało 🙂

Reply:
tworzyłam coś zamiast tego @lazy i zapomniałam skasować, zaraz wyślę poprawione